calendar_reboot.js

```diff
@@ -161,17 +188,13 @@ $(document).ready(function() {
         // and fill it with the information given in the form
         //required info are startDate and title. default to tracking 1 
         //year
+        
         var calendarTitle = $("#calendarTitle").val();
         var startDate = $("#startDate").val();
         var endDate = $("#endDate").val();
-            
         
-        var validateTitle = validateInput(calendarTitleId);
-        var validateStartDate = validateInput(startDateId);
-        var validateEndDate = validateInput(endDateId);
-        var validateDates = validateForm(startDate, endDate);
+        if (validateForm() && validateDates(startDate, endDate)) {
```

Comment:
Yes but I need to first check if inputs were given, then I need to check if the date inputs are valid. That's why I don't do them in the same function.